retox/__main__.py

```diff
@@ -97,18 +100,22 @@ def show_logs(screen, log_scene):
                 running = False
 
 
-def get_hashes(path, include={'.py'}):
+def get_hashes(path, exclude=None):
     '''
-    Get a dictionary of file paths and timestamps
+    Get a dictionary of file paths and timestamps.
+
+    Paths matching `exclude` regex will be excluded.
     '''
     out = {}
-    for root, _, files in os.walk(path):
-        path = root.split(os.sep)
-        for file in files:
-            for i in include:
-                if file.endswith(i):
-                    pytime = os.path.getmtime(os.path.join(root, file))
-                    out[os.path.join(root, file)] = pytime
+    for f in Path(path).rglob('*'):
```

Comment:
I've heard users having issues on Windows. Wonder if this fixes it. will be good to see the test case results on Window systems